datafusion/common/src/error.rs

```diff
@@ -150,6 +150,11 @@ pub enum SchemaError {
         qualifier: Box<TableReference>,
         name: String,
     },
+    /// Schema duplicate qualified fields with duplicate unqualified names
```

Comment:
changed to 
```rust
    /// Schema duplicate qualified fields with duplicate unqualified names. This is an error
    /// for schema in CREATE TABLE/VIEW statements, because the final object is going to retain
    /// and return the unqualified names only.
```